src/msg.fbs

```diff
@@ -182,10 +182,10 @@ table SetEnv {
 }
 
 table EnvironRes {
-  map: [EnvPair];
+  map: [KeyValue];
 }
 
-table EnvPair {
+table KeyValue {
```

Comment:
I wonder if it might be simpler to just store the key and value in one string, in the form `key=value`.

Afaik all operating systems maintain the invariant that the value starts after the first `=`. 
So this abstraction is actually somewhat broken in that if you have an = in the "key", it isn't actually the key. You can see this happen with node on both linux and windows:

```
piscisaureus@pumpkin:~$ node
>  require('child_process').execSync("echo $HELLO", { env: { 'HELLO=OOO': 'mars' }, stdio: 'inherit' })
OOO=mars
```

```
D:\denol>node
> require('child_process').execSync("echo %HELLO%", { env: { 'HELLO=OOO': 'mars' }, stdio: 'inherit' })
OOO=mars
```

Reply:
I don't think it's simpler. If we didn't have this massive serialization/deserialization framework, I might agree, but as it is, it's easier and probably more efficient like this.